src/System.Security.Cryptography.Encoding/tests/OidCollection.cs

```diff
@@ -89,5 +89,41 @@ public static void TestOidCollection()
 
         private const string SHA256_Name = "sha256";
         private const string SHA256_Oid = "2.16.840.1.101.3.4.2.1";
+
+        [Fact]
+        public void Enumerator_Success()
```

Comment:
It's kind of weird to have the consts in the middle.  Can you either move them to the top or bottom? (bottom would just be "move the new tests up a bit")